package.json

```diff
@@ -34,6 +34,7 @@
     "@uniswap/default-token-list": "^2.0.0",
     "@uniswap/permit2-sdk": "^1.2.0",
     "@uniswap/router-sdk": "^1.3.0",
+    "@uniswap/sdk-core": "^3.2.6",
```

Comment:
Revisit on adding the sdk-core explicitly since we already have transitive dependency.